server.py

```diff
@@ -70,10 +63,9 @@ def b64dec(s):
 
 
 def get_verified_email(token):
-    rsp = request.urlopen(''.join((
-        META['PORTIER_ORIGIN'],
-        '/.well-known/openid-configuration',
-    )))
+    rsp = request.urlopen(
```

Comment:
Here, too, I'd prefer the formatting changes (which I otherwise like for having one less line) in a separate commit, so that this one ist just about the config thingy.